torchx/test/torchx/device_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule Torchx.DeviceTest do
+  use ExUnit.Case, async: true
+
+  alias Torchx.Backend, as: TB
+
+  setup do
+    Nx.default_backend(Torchx.Backend)
+    :ok
+  end
+
+  describe "creation" do
+    test "from_binary" do
+      t = Nx.tensor([1, 2, 3], backend_options: [device: :cpu])
+      Nx.backend_transfer(t, TB, backend_options: [device: :cpu])
```

Comment:
For backend transfer, the options are already the backend options:

```suggestion
      Nx.backend_transfer(t, TB, [device: :cpu])
```

Btw, is there a way for us to ask Torch which devices are enabled?

Reply:
Yes, can check, but for CUDA only.